pathways/simulation.py

```diff
@@ -335,7 +361,10 @@ def run_simulation(
     totals.total_intercepted_pests /= float(num_simulations)
     totals.total_missed_pests /= float(num_simulations)
 
-    return totals
+    if detailed:
+        return details, totals
```

Comment:
Didn't you switch the order from last time? I think the `app.py` expects that the other way.

Reply:
Nice, fixed in app.py.